core/src/main/resources/migration/tenant/4.0-2019-06-14-OC-10998.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+	<changeSet author="jkeremian" id="2019-06-14-10998-01">
+		<addColumn tableName="discrepancy_note">
+			<column name="thread_uuid" type="VARCHAR(64)"/>
+			<column name="thread_number" type="int"/>
+		</addColumn>
+	</changeSet>
+
+	<changeSet author="jkeremian" id="2019-06-17-10998-02" >
+		<comment>insert thread_uuid and thread_number in discrepancy_note table</comment>
+		<sql splitStatements="false">
+			CREATE OR REPLACE FUNCTION insert_thread_info() RETURNS void AS
+			'DECLARE
+
+			counter INTEGER := 0;
+			uuid VARCHAR(50);
+			rec RECORD;
+
+			BEGIN
+			FOR rec IN select * from discrepancy_note  where thread_uuid isNull and discrepancy_note_type_id=3 and parent_dn_id isNull order by discrepancy_note_id
+			LOOP
+			SELECT md5(random()::text || clock_timestamp()::text)::uuid INTO uuid;
```

Comment:
You should use uuid_generate_v4() postgres function instead of this. You can check

Reply:
updated the method to call uuid_generate_v4() postgres function